lib/commands/smapi/smapi-command-handler.js

```diff
@@ -110,17 +110,20 @@ const smapiCommandHandler = (swaggerApiOperationName, flatParamsMap, commanderTo
     const beforeSendProcessor = new BeforeSendProcessor(inputCmdObj._name, paramsObject, modelIntrospector, profile);
     beforeSendProcessor.processAll();
 
-    // TODO revisit with debug flow
+    const functionName = _sdkFunctionName(swaggerApiOperationName);
+    const params = getParamNames(client[functionName]);
+    const args = _mapToArgs(params, paramsObject);
+
     if (inputOpts.debug) {
+        const payload = {};
```

Comment:
improving debug output to show how parameters are mapped to ask-smapi-sdk client